common/src/main/java/com/wynntils/core/net/UrlId.java

```diff
@@ -42,9 +42,9 @@ public enum UrlId {
     DATA_STATIC_LOOTRUN_TASKS_NAMED("dataStaticLootrunTasksNamed"),
     DATA_STATIC_MAPS("dataStaticMaps"),
     DATA_STATIC_MATERIAL_CONVERSION("dataStaticMaterialConversion"),
-    DATA_STATIC_PLACES("dataStaticPlaces"),
+    DATA_STATIC_MAPDATA_PLACES("dataStaticMapdataPlaces"),
     DATA_STATIC_SEASKIPPER_DESTINATIONS("dataStaticSeaskipperDestinations"),
-    DATA_STATIC_SERVICES("dataStaticServices"),
+    DATA_STATIC_MAPDATA_SERVICES("dataStaticMapdataServices"),
```

Comment:
Also, on these two, maybe we should try to keep a consistent naming scheme, so like `dataStaticServiceMapFeatures` (`dataStaticServiceMapLocations`, if that is what you prefer).